src/System.Diagnostics.Debug/tests/DebugTests.cs

```diff
@@ -3,12 +3,15 @@
 // See the LICENSE file in the project root for more information.
 
 using Xunit;
+using System.Runtime.InteropServices;
 
 namespace System.Diagnostics.Tests
 {
     public class DebugTests
     {
-        [Fact]
+        public static bool IsNotDesktopJob = !RuntimeInformation.FrameworkDescription.Contains(".NET Framework");
```

Comment:
"System.InvalidOperationException : An appropriate member "IsNotDesktopJob" could not be found. The conditional method needs to be a static method or property on this or any ancestor type, of any visibility, accepting zero arguments, and having a return type of Boolean."

Looks like it needs to be a property